.gitignore

```diff
@@ -30,3 +30,6 @@ dogstatsd.yaml
 
 # Ignore debs from the root of the project.
 datadog-agent*_amd64.deb
+
+# Ignore pem created during the tests
```

Comment:
during ~~the~~ tests

Reply:
Alternatively you could do some cleanup and delete the files in the tests tear-down

Reply:
I did also